liferay-workspace/ext/login-web-ext/gradlew

```diff
@@ -0,0 +1,169 @@
+#!/usr/bin/env bash
```

Comment:
We don't need to add gradle wrapper to workspace projects.

Reply:
removed